packages/lib/hooks/useTypedQuery.ts

```diff
@@ -86,5 +86,19 @@ export function useTypedQuery<T extends z.AnyZodObject>(schema: T) {
     }
   }
 
-  return { data: parsedQuery, setQuery, removeByKey, pushItemToKey, removeItemByKeyAndValue };
+  // Remove all query params from the URL
+  function removeAllQueryParams(keys: OutputOptionalKeys[]) {
```

Comment:
I noticed there is a non-optional function param but when you call this function, you don't pass any params.

Reply:
@joeauyeung Initially i had planned to create a function to remove multiple keys from the query param, then later on decided on the current approach and my bad i forgot to remove the unnecessary param